Will Heikoff Documentation

```diff
@@ -1,13 +1,34 @@
-Found zip code file for line 22: ZIPS.shp
+Will Heikoff Documentation
+Goal: To get the python script K-function Local, G:\MAX-Filer\Research\Labs-kbuzard-S18\kfunctions\K-function Local, for California R & D Labs running for 9 iterations
+
+6/12/18
+Searched for file which exhibits the zip code with the associated Total and manufacturing employment for the zip code.
+
+6/15/18
+Found zip code file for line 22: ZIPS.shp, using trial and error methods testing different files with the name ‘zips’ within the California folder, plugging the files into the ArcMap file ‘Base’, located within the same folder, to find a file which exhibits the zip code with the associated Total and manufacturing employment for the zip code.
+	Located at G:\MAX-Filer\Research\Labs-kbuzard-S18\CARR\California\Base Data
 Entered ZIPS.shp into line 22
-Found labs data for line 23: Labs.shp
+Found labs data for line 23: Labs.shp, using trial and error methods testing different files with the name ‘zips’ within the California folder, plugging the files into the ArcMap file ‘Base’, located within the same folder, to find a file which exhibits the Labs street address and associated ‘X’ and ‘Y’ cooridinates
+	Located at G:\MAX-Filer\Research\Labs-kbuzard-S18\CARR\California\Base Data
 Entered Labs.shp into line 23
-Entered ‘ZIP’ on Line 26 for variable name
-Entered variable names ‘X’ and ‘Y’ variable names into respective spots on line 47-48
+Entered ‘ZIP’ on Line 26 for variable name, after plugging in the ‘ZIPS.zhp’ file into ArcMap and using the attribute Table to establish what the zip code variable was
```

Comment:
ZIPS.zhp should be ZIPS.shp

Here, we want not the variable that identifies the zip code (which is what ZIP) is, but the data that we are using to establish the correct proportions for how many labs are randomized into each zip code. In this simpler process (we're going to add a layer once we figure this out: total employment by Census block), we're using manufacturing employment. So the right variable looks like 'Emp_Manufa' -- both the title and the fact that it's smaller than 'Emp_Total' bears that out.

Given the changeover to the scipy version of the functions, please go back and enter notes from the beginning -- just little signposts so that someone reading this would know it's coming up, and where you don't want to repeat the work you've done, noting the changes (like where these things are entered on different lines).